mqtt/mqtt-edgehub/src/settings.rs

```diff
@@ -216,6 +241,28 @@ mod tests {
 
     const DAYS: u64 = 24 * 60 * 60;
 
+    #[test]
```

Comment:
@huguesBouvier , can you please add #[serial] to this test, so it won't interfere with others.